speedy-links/src/speedy-links.ts

```diff
@@ -0,0 +1,99 @@
+export interface ActivateLinksOptions {
+  mapLinkUrlToModuleUrl: (url: string) => string | Promise<string>;
```

Comment:
´moduleLink’ ? since we already have moduleExecutor, api would look more coherent to me

Reply:
my logic behind the current naming?

LinkUrl
`http://localhost:8080/f9004900-9e3a-11ec-a43b-31caffe9627a/vite/4BeMe20hqOWTkdUL2NuJ/css-showcases/doc/css-showcases`

ModuleUrl
`http://localhost:8080/f9004900-9e3a-11ec-a43b-31caffe9627a/vite/4BeMe20hqOWTkdUL2NuJ/css-showcases/doc/css-showcases.render-doc.js`

coherent

I don't quite understand `moduleLink`

Reply:
this is a mapping function, having map inside the name looked always odd to me. The result expected is a moduleLink hence I prefer to name map function to what they are mapping

Reply:
so moduleUrl here I understand correctly

Reply:
link is a UI element or a connection with smth, while here it is a URL
why do you see it as a link?